mathjax3-ts/a11y/sre-node.ts

```diff
@@ -26,7 +26,7 @@ declare const require: (name: string) => any;
 
 const SRE = require('speech-rule-engine');
 
-declare var global: any;
+declare let global: any;
```

Comment:
Is `let` to be preferred to the original `const`?

Reply:
No. I played with `var`, forgot to take it out before I pushed and instead of resetting it to `const` I used `let`. Sorry about the confusion.